docs/manual/docs/cmd/spo/folder/folder-copy.md

```diff
@@ -43,6 +44,11 @@ Copies folder from a document library to another site in the same site collectio
 spo folder copy --webUrl https://contoso.sharepoint.com/sites/test1 --sourceUrl /Shared%20Documents/MyFolder --targetUrl /sites/test1/HRDocuments/
 ```
 
+Copies folder to a document library in another site collection. Allow for schema mismatch
+  ```sh
```

Comment:
There should be an empty line between text and a code block. Additionally, the code block shouldn't be indented.